ohsome/constants.py

```diff
@@ -5,3 +5,5 @@
 OHSOME_BASE_API_URL = "https://api.ohsome.org/v1/"
 DEFAULT_LOG = True
 DEFAULT_LOG_DIR = "./ohsome_log"
+# update version in pyproject.toml as well
+OHSOME_VERSION = "0.1.0rc1"
```

Comment:
please change this to 0.1.0.rc2

Reply:
I think you meant `0.1.0rc2`, or did you want to use a `.` as a separator between the version numbers and the _rc_ indicatior?

Reply:
ah yes, I meant 0.1.0rc2